Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rucio/rucio-clients:release-1.23.7.post1
+FROM rucio/rucio-clients:latest
```

Comment:
Building from `latest` is quite dangerous since we will always pick up new versions and not know the tag.

Reply:
I guess we could have the `develop` branch us this, but tie it down for the released versions